Clarify workspace recovery requirements (#2629)
hashicorp/web-unified-docs: content/terraform-docs-common/docs/cloud-docs/workspaces/create.mdx

@include 'beta.mdx'
 
You have 30 days to recover a deleted workspace. While a discarded workspace is recoverable, HCP Terraform and Terraform Enterprise continue to record new configuration versions committed through VCS webhooks, but commits don’t trigger new runs. You can’t recover a workspace that has the same name as an existing workspace. To proceed, you must either delete or rename the existing workspace. 

> You must be on a paid subscription to recover a workspace. Refer to [Workspace recovery requirements](#workspace-recovery-requirements) for more information.

Reviewer: ```suggestion
You must have a paid subscription to recover a workspace. Refer to [Workspace recovery requirements](#workspace-recovery-requirements) for more information.
```

Totally feel free to take or leave this suggestion, this is just a small tweak to match the wording in the requirements section.

Also, should this also be in an exclusion tag as it is in the requirements section?
